compiler-core/src/type_/environment.rs

```diff
@@ -497,11 +501,15 @@ impl<'a> Environment<'a> {
             .entity_usages
             .pop()
             .expect("Expected a bottom level of entity usages.");
-        self.handle_unused(unused);
+        self.handle_unused(unused.clone());
```

Comment:
I don't think we need to clone here? I can't see it being used afterwards

Reply:
You are right, thanks!